controllers/apartmentsController.js

```diff
@@ -1,11 +1,12 @@
+const auth = require('../middleware/auth')
 const {Apartment, validate} = require('../models/apartment');
 
 getAllApartments = async (req, res) => {
     const apartments = await Apartment.find().sort(typeof 'available' == true);
     res.send(apartments);
 }
 
-createApartment = async (req, res) => {
+createApartment = auth,async (req, res) => {
```

Comment:
This isn't clean. You need to add the auth middleware at the level of the route. For example:
```
 router.post('/', auth, controller...);
```